permits/templates/permits/permit_request_detail.html

```diff
@@ -35,37 +35,48 @@ <h1>
     <div class="row floating-elem">
       <div class="col">
         <dl>
-        <dd>
-          <h4>{% trans "Auteur-e" %}</h4>
-        </dd>
-        <dd>
-          <a href="mailto:{{ permit_request.author }} <{{ permit_request.author.user.email }}>" target="_blank" rel="noreferrer">
-            {{ permit_request.author.user.first_name }} {{ permit_request.author.user.last_name }} &lt;{{ permit_request.author.user.email }}&gt;
-          </a>
-        <dd>
-          {{ permit_request.author.address }} {{ permit_request.author.zipcode }} {{ permit_request.author.city }}
-        </dd>
-        <dd>  
-          {{ permit_request.author.phone_first }}  {{ permit_request.author.phone_second }}
-        </dd>
-        <dd>
-          {{ permit_request.author.company_name }} {{ permit_request.author.vat_number }}
-        </dd>
-        <dd>
-          {% if permit_request.validation_pdf %}
           <dd>
-            {% trans "Document délivré :" %}
+            <h4>{% trans "Auteur-e" %}</h4>
           </dd>
           <dd>
-              <i class="fa fa-download" aria-hidden="true"></i>
-              <a href="{{ permit_request.validation_pdf.url }}" target="_blank" rel="noreferrer">
-                Télécharger le document PDF
-              </a>
+            <a href="mailto:{{ permit_request.author }} <{{ permit_request.author.user.email }}>" target="_blank" rel="noreferrer">
+              {{ permit_request.author.user.first_name }} {{ permit_request.author.user.last_name }} &lt;{{ permit_request.author.user.email }}&gt;
+            </a>
+          </dd>
+          <dd>
+            {{ permit_request.author.address }}, {{ permit_request.author.zipcode }} {{ permit_request.author.city }}
+          </dd>
+          <dd>
+            {% if permit_request.author.phone_first %}
+            <a href="tel:{{ permit_request.author.phone_first }}" target="_blank" rel="noreferrer" title="{% trans 'Téléphone principal' %}">{{ permit_request.author.phone_first }}</a>
+            {% endif %}
+            {% if permit_request.author.phone_second %}
+            &nbsp;/&nbsp; <a href="tel:{{ permit_request.author.phone_second }}" target="_blank" rel="noreferrer" title="{% trans 'Téléphone secondaire' %}">{{ permit_request.author.phone_second }}</a>
```

Comment:
Any better way of adding space ?

Reply:
Any suggestion?

With just ` / `, I have:
![image](https://user-images.githubusercontent.com/4549666/147057346-a03f8897-134b-4962-97cb-3ee5cab47047.png)

Reply:
well, we won't use css for just this point, so, that's ok to go with `&nbsp`; I guess